### ISSUE TITLE
Quicksorts not working with hoppers? Maybe? 

### ISSUE THREAD

**Issue by Mattabase** [pcal43/quicksort#16]
Not sure if it's a bug or a feature, but the quicksort chest requires the player to open and close the chest for it to sort. It will not start sorting if you for example hopper items into the chest, and it was not already sorting.

**Comment by pcal43** [comment]
Yeah at the moment it's working-as-designed.  But it's not unreasonable to want the sorting to happen when a hopper inserts an item or maybe on a redstone pulse.

I had sort of wanted to keep it a simple inventory sorting aid; if it starts to turn into a full-on redstone component, there are more performance worries and more ways for things to go wrong.  But I'll give it some thought next time I'm working on it. 

Thanks!

**Comment by seriaati** [comment]
please add
